src/clr-angular/popover/dropdown/_dropdown.clarity.scss

```diff
@@ -37,11 +37,11 @@
       }
 
       &.btn {
-        padding-right: $clr-button-horizontal-padding + $clr-dropdown-caret-icon-dimension +
+        padding-right: $clr-btn-horizontal-padding + $clr-dropdown-caret-icon-dimension +
```

Comment:
Something looks wrong with the gemini tests and dropdowns, the > icon has moved. I'm not sure where this is defined.

Reply:
The carets in the nested dropdowns were not aligned properly. They are now. I made a note of this in the commit message...
> • fixed alignment issue with caret in nested dropdowns

Reply:
### BEFORE...
![Screen Shot 2019-03-12 at 2 20 46 PM](https://user-images.githubusercontent.com/2728359/54229409-32f13400-44d2-11e9-8a8d-9974cb91dd99.png)

### AFTER...
![Screen Shot 2019-03-12 at 2 22 44 PM](https://user-images.githubusercontent.com/2728359/54229495-6338d280-44d2-11e9-879d-401ee800aee5.png)

Notice the vertical alignment of the carets.